test/datastore-test.js

```diff
@@ -221,4 +221,68 @@ describe("datastore", () => {
       assert.strictEqual(result, ds);
     });
   });
+
+  describe("persistence", () => {
+    let cached = new Map();
+    let expectedID;
+    let something = {
+      title: "Sa Tuna2",
+      entry: {
+        kind: "login",
+        username: "foo",
+        password: "bar"
+      }
+    };
+
+    before(async () => {
+      localdatabase.startup();
+    });
+    after(async () => {
+      // cleanup databases
+      await localdatabase.teardown();
+    });
+
+    it("add a value to 1st datatore", async () => {
+      let main = new DataStore({
+        prompts: {
+          unlock: unlockWin
+        },
+        keys: loadEncryptedKeys()
+      });
+      main = await main.prepare();
+      await main.unlock;
+
+      let result = await main.add(something);
+      assert.deepNestedInclude(result, something);
+      cached.set(result.id, result);
+      let stored = await main.list();
+      checkList(stored, cached);
+
+      // result is the full item
+      let expected = result;
+      result = await main.get(expected.id);
+      assert(expected !== result);
+      assert.deepEqual(result, expected);
+
+      expectedID = expected.id;
+    });
+
+    it("data persists into second datastore", async () => {
+      let secondDatastore = new DataStore({
```

Comment:
As above, construct without any arguments.  This time, there should be no need for the `initialize()` call.